src/OpenTelemetry.Exporter.Jaeger/JaegerExporter.cs

```diff
@@ -40,7 +40,7 @@ internal class JaegerExporter : BaseExporter<Activity>
         private int batchByteSize;
         private bool disposedValue; // To detect redundant dispose calls
 
-        internal JaegerExporter(JaegerExporterOptions options, TTransport clientTransport = null)
+        public JaegerExporter(JaegerExporterOptions options, TTransport clientTransport = null)
```

Comment:
I think the `TTransport` is used for test/benchmarks. Could we do...

```csharp
public JaegerExporter(JaegerExporterOptions options) : this(options, null) { }
internal JaegerExporter(JaegerExporterOptions options, TTransport clientTransport = null) { ... }
```

That _should_ allows us to keep the Thrift stuff internal.

Reply:
@CodeBlanch - Thanks for the feedback. I have updated the PR - Please review.